src/CommonDBTM.php

```diff
@@ -6571,4 +6582,23 @@ public function isGlobal(): bool
 
         return $is_global;
     }
+
+    /**
+     * Default search request
+     *
+     * @return array
+     */
+    public static function getDefaultSearchRequest()
+    {
+        return [
+            'criteria' => [
+                [
+                    'link'       => 'and',
+                    'field'      => 'view',
```

Comment:
I see that the `view` criteria can indeed be disabled.

I feel like there is a conflict between `getDefaultCriteria` and `getDefaultSearchRequest`.
Only one should exist (probably `getDefaultSearchRequest` as it can be overidden by children classes).

Should I delete `getDefaultSearchRequest` and use `getDefaultCriteria` everywhere instead (with the correct `view` code taken from `getDefaultSearchRequest`) ?

Reply:
Most of plugins that are defining the `getDefaultSearchRequest` are just passing `sort`+`order` params, but no `criteria`.

If you define a `CommonDBTM::getDefaultSearchRequest()`, plugins will have to be updated to do something like:
```php
// PluginFormcreatorFormAnswer

    static function getDefaultSearchRequest()
    {
        return array_merge(
            parent::getDefaultSearchRequest(),
            [
                'sort'   => 6,
                'order' => 'DESC'
            ]
        );
    }
```

Sounds an acceptable change for me, but requires a changelog entry, and will probably be a pain for you to backport plugin patches on 10.0 version.

Reply:
I guess I'll keep it simple for this PR and just fix `getDefaultSearchRequest`.

Reply:
Done 2a925cd1b0.